src/resources.qrc

```diff
@@ -1,35 +1,16 @@
 <RCC>
     <qresource prefix="/translations">
         <file alias="translation_de">res/translation/translation_de_DE.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_fr">res/translation/translation_fr_FR.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_pt_PT">res/translation/translation_pt_PT.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_pt">res/translation/translation_pt_BR.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_es">res/translation/translation_es_ES.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_nl">res/translation/translation_nl_NL.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_it">res/translation/translation_it_IT.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_pl">res/translation/translation_pl_PL.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_sk">res/translation/translation_sk_SK.qm</file>
-    </qresource>
-    <qresource prefix="/translations">
         <file alias="translation_sv">res/translation/translation_sv_SE.qm</file>
     </qresource>
-
```

Comment:
I'm wondering if some newer version of Qt Creator than the advised "newest" (5.12.2) did this.  If so, please revert.

Reply:
Hmm ????
I have never touched that file and am using the same version of Qt since the end of last year.  Why it should suddenly decide to change something ????

Reply:
> I have never touched that file

You did:) .... when you added the two image files for the Fancy status LEDs with accessibility markers. Not sure if you did that manually or in Qt of course...

Reply:
You can also fix it like this:
```
git reset HEAD~
git add (all the other files)
git add -p src/resources.qrc
(n to everything except the two new png files)
git commit -m 'your commit message'
git checkout -- src/resources.qrc
```
(then, of course, test a local build before `git push --force`)

Reply:
I don't think there is a problem with this change. It just built happily on my Pi, and also on the Github CI. It is also more consistent with the sections for other resources.